db/migrate/20180321015659_create_rebates.rb

```diff
@@ -0,0 +1,13 @@
+class CreateRebates < ActiveRecord::Migration[5.1]
+  def change
+    create_table :rebate_forms do |t|
+      t.text :token
+      t.json :payload
```

Comment:
@dlouise64 i'm thinking we use the json column type to store much of the form data. At least for alpha. Then we can send almost any format. But some fields we might make a column for.